doc/10-Channels.md

```diff
@@ -1,4 +1,21 @@
-## Channel Plugin
+# Channels
+
+After Icinga Notifications decides to send a notification of any kind, it will be passed to a channel plugin.
+Such a channel plugin submits the notification event to a channel, e.g., email or a chat client.
+
+Icinga Notifications comes packed with channel plugins, but also enables you to develop your own plugins.
+
+To make those plugins available to Icinga Notifications, they must be placed in the [plugin channel directory](03-Configuration.md#channel-plugin-directory),
```

Comment:
> plugin channel directory

Well that's a new name, I don't think it's called like this anywhere else.

Reply:
Just another permutation of the channel directory plugin ^W^W^W directory plugin channel ^W^W^W  ... :upside_down_face: 

With #225 it got "simplified" to `channels-dir` and I have reflected this change here. Thanks.